lib/src/main/java/com/iwillfailyou/javaparser/NodeDescription.java

```diff
@@ -59,7 +57,12 @@ public NodeDescription(
     public String asString() {
         final StringBuilder description = new StringBuilder();
         description.append(new NodePath(node).asString());
-        final Optional<Range> range = node.getRange();
+        final Optional<Range> range = node.getChildNodes()
+            .stream()
+            .filter(aNode -> !(aNode instanceof MarkerAnnotationExpr))
```

Comment:
I don't really like the instanceof but I couldn't find a nicer way to get the pointer of the method declaration.